src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -623,6 +644,41 @@ describe('SavedObjectsRepository', () => {
       expect(onBeforeWrite).toHaveBeenCalledTimes(1);
     });
 
+    it('accepts a custom refresh setting', async () => {
+      callAdminCluster.mockReturnValue({
+        items: [
+          { create: { type: 'config', id: 'config:one', _primary_term: 1, _seq_no: 1 } },
+          { create: { type: 'index-pattern', id: 'config:two', _primary_term: 1, _seq_no: 1 } },
+        ],
+      });
+
+      await savedObjectsRepository.bulkCreate([
+        {
+          type: 'config',
+          id: 'one',
+          attributes: { title: 'Test One' },
+          references: [{ name: 'ref_0', type: 'test', id: '1' }],
+        },
+        {
+          type: 'index-pattern',
+          id: 'two',
+          attributes: { title: 'Test Two' },
+          references: [{ name: 'ref_0', type: 'test', id: '2' }],
+        },
+      ], {
+        refresh: true
+      });
+
+      expect(callAdminCluster).toHaveBeenCalledTimes(1);
+      const bulkCalls = callAdminCluster.mock.calls.filter(([path]) => path === 'bulk');
```

Comment:
nit: Not sure why we're filtering in some other tests, but this seems unecessary